<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIRenameName</class>
 <widget class="QWidget" name="UIRenameName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="m_pqcbReplaceName">
     <property name="text">
      <string>Replace Name With</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="IComQLineEdit" name="m_pqleReplaceName">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_pqlblChangeCase">
       <property name="text">
        <string>Change Case</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_pqcboChangeCase">
       <item>
        <property name="text">
         <string>No Change</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Title Case</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sentance case</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lower case</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UPPER CASE</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="m_pqcbReplaceTheText">
     <property name="text">
      <string>Replace The Text</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="IComQLineEdit" name="m_pqleReplaceTheText">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="m_pqlblReplaceTheTextWith">
     <property name="text">
      <string>With</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="IComQLineEdit" name="m_pqleReplaceTheTextWith">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="m_pqcbInsertTheText">
     <property name="text">
      <string>Insert The Text</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="IComQLineEdit" name="m_pqleInsertTheText">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="m_pqlblInsertTheTextAtPos">
     <property name="text">
      <string>At Position</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="IComQLineEdit" name="m_pqleInsertTheTextAtPos">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="m_pqcbInsertAtStart">
     <property name="text">
      <string>Insert At Start</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="IComQLineEdit" name="m_pqleInsertAtStart">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="m_pqcbInsertAtEnd">
     <property name="text">
      <string>Insert At End</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="IComQLineEdit" name="m_pqleInsertAtEnd">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="m_pqcbCropAtPos">
     <property name="text">
      <string>Crop At Position</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="IComQLineEdit" name="m_pqleCropAtPos">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="m_pqlblCropAtPosNextNChar">
     <property name="text">
      <string>The Next N Characters</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="IComQLineEdit" name="m_pqleCropAtPosNextNChar">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="m_pqcbLeftCropNChar">
     <property name="text">
      <string>Left Crop N Characters</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="IComQLineEdit" name="m_pqleLeftCropNChar">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCheckBox" name="m_pqcbRightCropNChar">
     <property name="text">
      <string>Right Crop N Characters</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="IComQLineEdit" name="m_pqleRightCropNChar">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IComQLineEdit</class>
   <extends>QLineEdit</extends>
   <header>IComQLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_pqcbReplaceName</tabstop>
  <tabstop>m_pqleReplaceName</tabstop>
  <tabstop>m_pqcbReplaceTheText</tabstop>
  <tabstop>m_pqleReplaceTheText</tabstop>
  <tabstop>m_pqleReplaceTheTextWith</tabstop>
  <tabstop>m_pqcbInsertTheText</tabstop>
  <tabstop>m_pqleInsertTheText</tabstop>
  <tabstop>m_pqleInsertTheTextAtPos</tabstop>
  <tabstop>m_pqcbInsertAtStart</tabstop>
  <tabstop>m_pqleInsertAtStart</tabstop>
  <tabstop>m_pqcbInsertAtEnd</tabstop>
  <tabstop>m_pqleInsertAtEnd</tabstop>
  <tabstop>m_pqcbCropAtPos</tabstop>
  <tabstop>m_pqleCropAtPos</tabstop>
  <tabstop>m_pqleCropAtPosNextNChar</tabstop>
  <tabstop>m_pqcbLeftCropNChar</tabstop>
  <tabstop>m_pqleLeftCropNChar</tabstop>
  <tabstop>m_pqcbRightCropNChar</tabstop>
  <tabstop>m_pqleRightCropNChar</tabstop>
  <tabstop>m_pqcboChangeCase</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_pqcbReplaceName</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleReplaceName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbReplaceName</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleReplaceName</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbReplaceTheText</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleReplaceTheText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbReplaceTheText</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleReplaceTheText</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbReplaceTheText</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleReplaceTheTextWith</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbInsertTheText</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleInsertTheText</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbInsertTheText</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleInsertTheText</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbInsertTheText</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleInsertTheTextAtPos</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbInsertAtStart</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleInsertAtStart</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbInsertAtStart</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleInsertAtStart</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbInsertAtEnd</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleInsertAtEnd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbInsertAtEnd</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleInsertAtEnd</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbCropAtPos</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleCropAtPos</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbCropAtPos</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleCropAtPos</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbCropAtPos</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleCropAtPosNextNChar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbLeftCropNChar</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleLeftCropNChar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbLeftCropNChar</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleLeftCropNChar</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbRightCropNChar</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleRightCropNChar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbRightCropNChar</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleRightCropNChar</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
